apicast/src/policy/cors.lua

```diff
@@ -0,0 +1,95 @@
+--- CORS policy
+-- This policy enables CORS (Cross Origin Resource Sharing) request handling.
+-- The policy is configurable. Users can specify the values for the following
+-- headers in the response:
+--   - Access-Control-Allow-Headers
+--   - Access-Control-Allow-Methods
+--   - Access-Control-Allow-Origin
+--   - Access-Control-Allow-Credentials
+-- By default, those headers are set so all the requests are allowed. For
+-- example, if the request contains the 'Origin' header set to 'example.com',
+-- by default, 'Access-Control-Allow-Origin' in the response will be set to
+-- 'example.com' too.
+
+local concat = table.concat
+
+local policy = require('policy')
```

Comment:
This will produce a deprecation warning once https://github.com/3scale/apicast/pull/486 is merged. No action needed here, but that PR is going to have to fix this.